src/main/java/wolox/training/repositories/UserRepository.java

```diff
@@ -5,4 +5,5 @@
 
 public interface UserRepository extends CrudRepository<User, Long> {
     User findFirstByName(String name);
+    User findByUsername(String name);
```

Comment:
Remember to return a wrapped User in an Optional, instead of a User directly  :)